src/drt/CMakeLists.txt

```diff
@@ -223,3 +223,33 @@ if (VTune_FOUND)
   )
 
 endif(VTune_FOUND)
+
+if (Python3_FOUND AND BUILD_PYTHON)
+  swig_lib(NAME          drt_py
+           NAMESPACE     drt
+           LANGUAGE      python
+           I_FILE        src/TritonRoute-py.i
+           SWIG_INCLUDES ${PROJECT_SOURCE_DIR}/include/drt
+           SCRIPTS       ${CMAKE_CURRENT_BINARY_DIR}/drt_py.py
+  )
+
+  target_include_directories(drt_py
+    PUBLIC
+      include
+  )
+
+  target_link_libraries(drt_py
```

Comment:
I thought we could have drt_py depend on drt rather than duplicating the dependencies.

Reply:
So I am not certain how this would work  in cmake. Can you give me an example?

Reply:
Actually, my first guess was correct. So this is done.

Reply:
You did this previously in https://github.com/The-OpenROAD-Project/OpenROAD/blob/6052d6a75bfbb6308f4f70820388afd34123407f/src/dpl/CMakeLists.txt#L96